src/slack_notify.ml

```diff
@@ -66,16 +70,22 @@ let execute token username channel icon_url icon_emoji msg =
   let channel = Slacko.channel_of_string channel in
   let chat = Slacko.Channel channel in
   let msg = Slacko.message_of_string msg in
+  let attachments =
```

Comment:
It would. My focus was on getting the functionality working so I could get some feedback on the general approach, so there's probably a bunch of stuff like this that I can do fairly easily. :-)